packages/jaeger-ui/src/reducers/trace.js

```diff
@@ -114,7 +115,7 @@ function searchDone(state, { meta, payload }) {
   }
   const traces = { ...state.traces, ...resultTraces };
   const search = { ...state.search, results, state: fetchedState.DONE };
-  return { ...state, search, traces };
+  return { ...state, search, traces, tracesToDownload: payloadData};
```

Comment:
nit: I don't think this reducer knows _why_ it's returning the full payload, so naming the output `tracesToDownload` is not accurate. Can we name it something like `rawTraces`?

Reply:
Good point. I will do it.